protocol.md

```diff
@@ -726,6 +727,120 @@ Upload-Length: 11
 Upload-Concat: final;/files/a /files/b
 ```
 
+### Client Tag
```

Comment:
As @smatsson said, this extension should probably be called Upload Tag as the header.

Reply:
@Acconut It's a bit longer, but maybe go for `Upload-Client-Tag` as header name and leave the extension name as "Client tag"? This would be consistent with other extensions e.g. `checksum` -> `Upload-Checksum`, `concatenation` -> `Upload-concat`. What do you think?

Reply:
That's a fair point. I don't really have a strong opinion either way so Upload-Tag sounds good to me.

Reply:
Ok, I'll change the name to Upload Tag with the next commit.